.mailmap

```diff
@@ -0,0 +1,20 @@
+Chris Chiu <chiu@endlessos.org> <chiu@endlessm.com>
+Cosimo Cecchi <cosimo@endlessm.com> <cosimoc@gnome.org>
+Dan Nicholson <dbn@endlessos.org> <nicholson@endlessm.com>
+Derek Foreman <derek@endlessm.com>
+Devin Ekins <devinj.ekins@gmail.com>
+Devin Ekins <devinj.ekins@gmail.com> <DevinJ.Ekins@gmail.com>
+Kurt von Laven <kurt@endlessm.com>
```

Comment:
You can map me to kurt.von.laven@gmail.com since this email no longer exists. Thanks for helping maintain this project!

Reply:
Hi, great to hear from you! I think we missed overlapping at Endless by just a couple of months.

Reply:
Bad luck!

Reply:
As a thinly veiled excuse to meet, I am happy to share everything I think I did wrong/would have done differently in retrospect If the metrics system ever undergoes a major refactor.

Reply:
That'd be great – I'll drop you an email!